inspirehep/modules/migrator/tasks.py

```diff
@@ -267,6 +267,7 @@ def _get_records_to_update_generator(citations_lookup):
                     '_index': index,
                     '_type': doc_type,
                     '_id': str(uuid),
+                    '_version_type': 'force',  # XXX: removed in ES@5
```

Comment:
This is not enough: we also need to pass the current version in order for it to be preserved.